src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -139,14 +139,40 @@ public static void RegisterInstance(VisualStudioInstance instance)
         /// </param>
         public static void RegisterMSBuildPath(string msbuildPath)
         {
-            if (string.IsNullOrWhiteSpace(msbuildPath))
+            RegisterMSBuildPath(new string[] {
+                msbuildPath
+#if NET46
```

Comment:
I think this needs an if-not-mono, right?

Reply:
I agree, but it's worth noting that without it, it would just do an extra check without actually failing.

Reply:
Also occurred to me that we don't support mono quite yet, but that's changing soon.